frontend/src/authentication/ProfilePage.jsx

```diff
@@ -93,10 +93,14 @@ function ProfilePage() {
   function handleCheckbox(event, setValue) {
     const value = event.target.value;
     setValue((prev) => {
-      if (prev.includes(value)) {
+      if (prev?.includes(value)) {
         return prev.filter((val) => val !== value);
       } else {
-        return [...prev, value];
+        if (prev) {
+          return [...prev, value];
+        } else {
+          return [value];
+        }
```

Comment:
Instead of making a new nested if we can keep it as a single level:

```
if(prev?...){
...
} else if (prev) {
...
} else {
...
}
```

Reply:
Okay, I will make the fix

Reply:
Fixed.